command/agent/node_endpoint.go

```diff
@@ -101,19 +101,21 @@ func (s *HTTPServer) nodeToggleDrain(resp http.ResponseWriter, req *http.Request
 		return nil, CodedError(405, ErrInvalidMethod)
 	}
 
-	// Get the enable value
-	enableRaw := req.URL.Query().Get("enable")
-	if enableRaw == "" {
-		return nil, CodedError(400, "missing enable value")
-	}
-	enable, err := strconv.ParseBool(enableRaw)
-	if err != nil {
-		return nil, CodedError(400, "invalid enable value")
+	var drainRequest api.NodeUpdateDrainRequest
```

Comment:
This breaks backward compat by not supporting `enable` anymore. We could maintain it just by keeping the old query param parsing code.